fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/AbstractResource.java

```diff
@@ -97,7 +98,12 @@
     @Optional
     protected Supplier<String> pidMinter;
 
-    protected UniqueValueSupplier defaultPidMinter = new DefaultPathMinter();
+    // TODO: Allow configuration properties to override to length and count
+    final int desiredLength = HierarchicalIdentifierSupplier.DEFAULT_LENGTH;
+    final int desiredCount = HierarchicalIdentifierSupplier.DEFAULT_COUNT;
```

Comment:
You can remove these. They are assumed if you instantiate the class without any arguments. So
```
protected UniqueValueSupplier defaultPidMinter = new ConfigurableHierarchicalSupplier(2, 4);
```
uses the value you pass and 
```
protected UniqueValueSupplier defaultPidMinter = new ConfigurableHierarchicalSupplier();
```
will use the constructor you will make from `protected` to `public` below which assumes the values from `HierarchicalIdentifierSupplier`.